### DISCUSSION TITLE
Support for stream URLs (radio, etc)

### DISCUSSION THREAD

**Discussion by fernplant** [music-assistant/support#763] [Feature requests and ideas]
I am unfamiliar with the format of streaming URLs but perhaps this could be added. For instance, my local public radio KQED has a streaming URL of https://streams.kqed.org/kqedradio

**Comment by marcelveldt** [comment]
Someday in the future I will create a nice GUI to add these URL's to MA manually but in the meanwhile there's a little trick:

Create an empty text file in your music directory and give it the m3u extension.
Paste your stream URL's in there, just one URL per line is all fine.
Reload MA/HA or just issue a manual sync.

Check your playlists section, there should be a playlist with the name you gave above.
Your streams will be there. Just click them and you should even be able to add them to the library....

**Comment by OzGav** [comment]
Please continue any further discussion in the existing thread https://github.com/music-assistant/hass-music-assistant/discussions/633
